pkg/porter/credentials.go

```diff
@@ -140,7 +210,11 @@ func (p *Porter) GenerateCredentials(opts CredentialOptions) error {
 		return nil
 	}
 	err = p.Credentials.Save(*cs)
-	return errors.Wrapf(err, "unable to save credentials")
+	if err != nil {
```

Comment:
I'm not sure why you changed this line? `Wrapf` returns nil when err is nil.

Reply:
My Bad